tests/unit/generators/conf_files/test_create_restmap_conf.py

```diff
@@ -225,6 +197,21 @@ def test_restmap_endpoints(global_config_all_json, input_dir, output_dir, ta_nam
         handleractions = list, create, delete, edit
         handlerpersistentmode = true
         """
+    ).lstrip()
+
+    restmap_conf = RestMapConf(
+        global_config_logging_with_user_defined_handlers,
+        input_dir,
+        output_dir,
+        addon_name=ta_name,
+        ucc_dir=UCC_DIR,
     )
+    file_paths = restmap_conf.generate()
+
+    assert file_paths is not None
+    assert file_paths.keys() == {"restmap.conf"}
+
+    with open(file_paths["restmap.conf"]) as fp:
+        content = fp.read()
 
     assert content == expected_top + expected_content

```

Comment:
Since we have split the test case, I think the expected_content could be merged into one variable.

Reply:
good catch, thanks, updated